src/renderers/WebGLRenderer.js

```diff
@@ -160,6 +164,11 @@ function WebGLRenderer( parameters ) {
 	let _clippingEnabled = false;
 	let _localClippingEnabled = false;
 
+	// transmission
+
+	let _transmissionRenderTarget = null;
+	const _transmissionSamplerSize = new Vector2( 1024, 1024 ); // Should be configurable?
```

Comment:
Shouldn't this be the same as the output render dimensions? Without refraction, the transmission lines up exactly with the viewport.